doc/nrf/libraries/bluetooth_services/services/ancs_client.rst

```diff
@@ -65,11 +76,24 @@ Use :c:func:`bt_ancs_notification_action` to make the Notification Provider perf
     Application<<=ANCS_Client  [label = "data_source callback(attr_response)"];
     |||;
 
+Samples using the library
+*************************
+The following |NCS| sample uses this library:
+
+* :ref:`peripheral_ancs_client`
+
+Dependencies
+************
+
+There are no dependencies for using this library.
+
 API documentation
 *****************
 
 | Header file: :file:`include/bluetooth/services/ancs_client.h`
-| Source files: :file:`subsys/bluetooth/services/ancs_client.c` and :file:`subsys/bluetooth/services/ancs_attr_parser.c` and :file:`subsys/bluetooth/services/ancs_app_attr_get.c`
+| Source files: :file:`subsys/bluetooth/services/ancs_client.c`,
+|               :file:`subsys/bluetooth/services/ancs_attr_parser.c`,
+|               :file:`subsys/bluetooth/services/ancs_app_attr_get.c`
```

Comment:
This does not seem to make the list of source files any prettier. You could maybe use a bullet list.